cmake/onnxruntime_common.cmake

```diff
@@ -88,9 +88,12 @@ if (onnxruntime_USE_MIMALLOC_STL_ALLOCATOR OR onnxruntime_USE_MIMALLOC_ARENA_ALL
     endif()
 endif()
 
-onnxruntime_add_include_to_target(onnxruntime_common date_interface safeint_interface wil)
-target_include_directories(onnxruntime_common PRIVATE ${CMAKE_CURRENT_BINARY_DIR} ${ONNXRUNTIME_ROOT}
-        ${eigen_INCLUDE_DIRS})
+onnxruntime_add_include_to_target(onnxruntime_common date_interface wil)
+target_include_directories(onnxruntime_common
+    PRIVATE ${CMAKE_CURRENT_BINARY_DIR} ${ONNXRUNTIME_ROOT} ${eigen_INCLUDE_DIRS}
```

Comment:
onnxruntime_add_include_to_target should already do it?

https://github.com/microsoft/onnxruntime/blob/master/cmake/CMakeLists.txt#L465